css/css-masking/clip-path/clip-path-round-zero-size.html

```diff
@@ -6,6 +6,8 @@
 <link rel="author" title="Emilio Cobos Álvarez" href="mailto:emilio@crisal.io">
 <link rel="author" title="Mozilla" href="https://mozilla.org">
 <link rel="match" href="clip-path-round-zero-size-ref.html">
+<!-- Allow differences of antialised pixels along rounded edges -->
+<meta name="fuzzy" content="0-1;0-2">
```

Comment:
If that's allowed, then it seems fair to increase the bounds to the limits that Safari shows on wpt.fyi, that is, 0-64;0-28.  (Fundamentally, it's the same "differences of antialiased pixels along rounded edges".

See https://wpt.fyi/results/css/css-masking/clip-path/clip-path-round-zero-size.html?label=experimental&label=master

Reply:
Absolutely, thanks for the suggestion. I didn't realize Safari was encountering the same issue, so I'll make that change.